modules/core/src/main/java/org/apache/ignite/mxbean/ClusterMetricsMXBean.java

```diff
@@ -260,6 +260,10 @@
     @Override @MXBeanDescription("Current PME duration in milliseconds.")
     public long getCurrentPmeDuration();
 
+    /** {@inheritDoc} */
+    @Override @MXBeanDescription("Maximum amount of nodes available for safe stop - stop without data loss.")
```

Comment:
I'd mention in metric description that caches with backups=0 are not reflected in the metric.

Reply:
Fixed.